Source/ui_android/java/com/virtualapplications/play/MainActivity.java

```diff
@@ -260,8 +243,8 @@ protected void onActivityResult(int requestCode, int resultCode, Intent data) {
 
 	private void displayAboutDialog()
 	{
-		long buildDate = getBuildDate(this);
-		String buildDateString = new SimpleDateFormat("yyyy/MM/dd", Locale.getDefault()).format(buildDate);
+		Date buildDate = BuildConfig.buildTime;
+		String buildDateString = new SimpleDateFormat("d MMM yyyy hh:mm a", Locale.getDefault()).format(buildDate);
```

Comment:
the hours was meant to allow you to identify different release on the same day... or are you referring purely to the date format/structure?